.github/workflows/lint.yml

```
@@ -0,0 +1,58 @@
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 50
+
+      - name: Initial setup
+        run: |
+          git config --global --add safe.directory "$PWD"
+          git fetch -fu origin develop:develop
+        shell: bash
+
+      - name: Run linters
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2155:warning:6:10: Declare and assign separately to avoid masking return values

(shellcheck)